File: src/boiler/erlang/day2.erl
-module(day2).
-export([day_two/0]).


hello(0) -> 
    0;
hello(Num) ->
    io:fwrite("hello, world~n"),
    hello(Num-1).

day_two() ->
    {ok, [Num]} = io:fread("", "~d"),
    hello(Num).

File: src/boiler/elixir/day2.exs
defmodule ElixirDay2 do
    def write_hello(num) when 
        is_integer(num) and num > 0 do
        IO.puts ("hello world")
        write_hello(num-1)
    end
    def write_hello(_) do
        :ok
    end
    def daytwo do
        IO.puts "Hello yo"
        num = String.to_integer(String.trim(IO.gets ("Enter number")))      
        write_hello(num)
    end
end
